<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>505</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>348</width>
    <height>505</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>348</width>
    <height>505</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Page de connexion</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(2, 20, 56);
background-color: rgb(8, 26, 62);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>330</width>
       <height>450</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>330</width>
       <height>500</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="connectPage">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="connectPageVerticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="mainAppLogoLabelPage1">
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>300</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>../../ressources/bestBettor logo.jpg</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="idLineEdit">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
           </property>
           <property name="placeholderText">
            <string>Identifiant (e-mail)</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="passwordLineEdit">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Mot de passe</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="seePasswordPushButton">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
             </property>
             <property name="text">
              <string>Voir</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="forgottenPasswordPushButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>126</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <underline>true</underline>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
border: 2px solid rgb(8, 26, 62);
border-radius: 7px;</string>
           </property>
           <property name="text">
            <string>Mot de passe oubli√©</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="connectPushButton">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
           </property>
           <property name="text">
            <string>Connexion</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="registrationPushButton">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
             </property>
             <property name="text">
              <string>Inscription</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="registerPage">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <layout class="QVBoxLayout" name="registerPageVerticalLayout">
         <item>
          <widget class="QPushButton" name="backPushButtonPage2">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
           </property>
           <property name="text">
            <string>&lt;- Retour</string>
           </property>
           <property name="autoRepeatInterval">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="mainAppLogoLabelPage2">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>../../ressources/bestBettor logo.jpg</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="pseudoLineEditPage2">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
           </property>
           <property name="placeholderText">
            <string>Nom d'affichage</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="idLineEditPage2">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>E-mail</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLineEdit" name="passwordLineEditPage2">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Mot de passe</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="seePasswordPushButtonPage2">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
             </property>
             <property name="text">
              <string>Voir</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QSpinBox" name="ageSpinBoxPage2">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
             </property>
             <property name="suffix">
              <string> ans</string>
             </property>
             <property name="prefix">
              <string>J'ai </string>
             </property>
             <property name="minimum">
              <number>15</number>
             </property>
             <property name="maximum">
              <number>122</number>
             </property>
             <property name="value">
              <number>15</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="acceptAgeCheckBoxPage2">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Je certifie avoir plus de 15 ans</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="acceptPushButtonPage2">
           <property name="maximumSize">
            <size>
             <width>225</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <underline>true</underline>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
border: 2px solid rgb(8, 26, 62);
border-radius: 7px;</string>
           </property>
           <property name="text">
            <string>Je consulte les conditions d'utilisation</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="acceptCheckBoxPage2">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>J'ai lu et j'accepte l'utilisation et l'enregistrement de
mes donn√©es selon la r√©glementation de la RGPD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="registrationPushButtonPage2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
           </property>
           <property name="text">
            <string>Je m'inscris</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ForgottenPassword">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QPushButton" name="backPushButtonPage3">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
         </property>
         <property name="text">
          <string>&lt;- Retour</string>
         </property>
         <property name="autoRepeatInterval">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="mainAppLogoLabelPage3">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>300</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>300</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>../../ressources/bestBettor logo.jpg</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="infoLabelPage3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Saisissez l'adresse e-mail de votre compte</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="emailLineEditPage3">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhEmailCharactersOnly</set>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Adresse e-mail du compte</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="emailPushButtonPage3">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 7px;</string>
         </property>
         <property name="text">
          <string>Envoyer l'e-mail de r√©cup√©ration</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>passwordLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>connectPushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
